packages/compass-e2e-tests/helpers/commands/set-ace-value.ts

```diff
@@ -36,6 +37,10 @@ export async function setAceValue(
 
   await clipboard.write(value);
 
+  expect(await clipboard.read()).to.equal(value);
```

Comment:
Using expect here as an assertion to try and get to the bottom of why this was doing nothing sometimes on linux. Either this await or the pause (or both?) seems to have fixed it.

Reply:
This reminds me of a weird keychain issue we stumbled upon quite awhile where if you don't first try to write any value to the keychain before running the app, it would fail to read anything from the app in CI 😅